drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/question_twenty_six.rb

```diff
@@ -326,6 +329,17 @@ class QuestionTwentySix < Base
       sub_populations
     end
 
+    private def ch_household_categories
+      # [ch_clause, row_index, remove_previously_counted]
+      {
+        'Chronically Homeless' => [a_t[:chronically_homeless].eq(true), 0, true],
+        'Client Doesn’t Know/Client Refused' => [a_t[:prior_living_situation].in([8, 9]), 2, true],
+        'Data Not Collected' => [a_t[:prior_living_situation].eq(99), 3, true],
+        'Not Chronically Homeless' => [a_t[:chronically_homeless].eq(false), 1, true], # Not chronically homeless if is none of the above
+        'Total' => [Arel.sql('1=1'), 4, false],
+      }.freeze
+    end
+
     private def ch_categories
       {
         'Chronically Homeless' => a_t[:chronically_homeless].eq(true),
```

Comment:
Let's double check the spec on this, to confirm the order doesn't need to match any specific order.